playground/package/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
These days you create new example apps with `create-vite`, not `create-react-app` :).